packages/cdk/utils/vNode.ts

```diff
@@ -45,7 +45,7 @@ export function isValidElementNode(node: VNodeChild): boolean {
  * @param key key of slots, default is 'default'
  * @param options the property of the render function
  */
-export function getSlotNodes(slots: Slots, key = 'default', options: unknown[] = []): VNode[] {
+export function getSlotNodes(slots: Slots, key = 'default', options = {}): VNode[] {
```

Comment:
这里搞错了，就是一个数组，我查了它是以arguments的形式来获取的

Reply:
` export type Slot = (...args: any[]) => VNode[] `
Slot 的类型定义是这样子的，实际上使用都是传一个 object 进去的，目前还没有看到有传第二个参数的场景。后面有的话，也应该改成。
```suggestion
export function getSlotNodes(slots: Slots, key = 'default', ...options: unknown[]): VNode[] {
```

Reply:
👍

Reply:
我想了一下，options还是一个数组，但是 `slots[key](...options)` 这样传进去更好一些吧，避免 `getSlotNodes` 这个方法的参数太多